### PR TITLE
chore: Bump Docker image golang to 1.21.5

### PR DESCRIPTION
**Pull request by alfrunes**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -36,24 +36,9 @@ RUN if [ "$TARGETARCH" = "arm64" ]; then CC=aarch64-linux-gnu-gcc && CC_FOR_TARG
...
-COPY --from=builder /usr/lib/aarch64-linux-gnu/libcrypto.so.1.1 /usr/lib/aarch64-linux-gnu/libcrypto.so.1.1
-COPY --from=builder /lib/aarch64-linux-gnu/libpthread.so.0 /lib/aarch64-linux-gnu/libpthread.so.0
-COPY --from=builder /lib/x86_64-linux-gnu/libpthread.so.0 /lib/x86_64-linux-gnu/libpthread.so.0
-COPY --from=builder /lib/aarch64-linux-gnu/libdl.so.2 /lib/aarch64-linux-gnu/libdl.so.2
-COPY --from=builder /lib/x86_64-linux-gnu/libdl.so.2 /lib/x86_64-linux-gnu/libdl.so.2
-COPY --from=builder /lib/x86_64-linux-gnu/libnss_dns.so.2 /lib/x86_64-linux-gnu/libnss_dns.so.2
-COPY --from=builder /lib/aarch64-linux-gnu/libnss_dns.so.2 /lib/aarch64-linux-gnu/libnss_dns.so.2
-COPY --from=builder /lib/x86_64-linux-gnu/libresolv.so.2 /lib/x86_64-linux-gnu/libresolv.so.2
-COPY --from=builder /lib/aarch64-linux-gnu/libresolv.so.2 /lib/aarch64-linux-gnu/libresolv.so.2
+COPY --from=builder /lib64 /lib64
```
Comment (oldgiova) [inline Dockerfile:39]: I've already tried to copy the whole directory, but this didn't work out (or at least made the image much bigger). 
feel free to try, though.

Comment (alfrunes) [reply Dockerfile]: The size indeed doubled from copying all the base directories (I guess the COPY instruction does not preserve symlinks). I refactored the build command to only include the required libraries and squashed all the COPY instructions into one.

**Comment by alfrunes** [comment]
@mender-test-bot start pipeline

File: Dockerfile (hunk 1 above)
Comment (oldgiova) [reply Dockerfile]: This doesn't work unfortunately: here's my raspberry test:

```
docker run --rm --name test registry.gitlab.com/northern.tech/mender/deployments:1111139775

/usr/bin/deployments: error while loading shared libraries: liblzma.so.5: cannot open shared object file: No such file or directory
```

**Comment by alfrunes** [comment]
@mender-test-bot start pipeline --fast

**Pull request merged**